test/playwright.spec.js

```diff
@@ -198,7 +195,7 @@ module.exports.addPlaywrightTests = ({testRunner, platform, products, playwright
           await state.tearDown();
         });
 
-        describe('', function() {
+        pageEnv = testRunner.environment('PageTest', () => {
```

Comment:
I'd expect pageEnv to nest in contextEnv

Reply:
It can, but there is no real need for that since they recreate context every time anyway.